cspell.json

```diff
@@ -90,6 +91,8 @@
     "numbervisibilityprocess",
     "OAUTHCONFIG",
     "oidc",
+    "overridability",
+    "overridable",
```

Comment:
no such word: https://dictionary.cambridge.org/spellcheck/english/?q=overridable

Reply:
maybe we could go with simple language that is understood by most, rather than some terms that will look overengineered for other devs? just thinking aloud

Reply:
For now leave it as it is please. "Overridable" is adjective, what we need here is a noun.

Reply:
just to be clear
old word -> new word

overridable -> overridable (same)
overridability -> override

is it ok or should I revert the change? @popovicsandras